util/libcrypto.num

```diff
@@ -5683,8 +5683,13 @@ OSSL_TARGETING_INFORMATION_free         ?	3_4_0	EXIST::FUNCTION:
 OSSL_TARGETING_INFORMATION_new          ?	3_4_0	EXIST::FUNCTION:
 OSSL_TARGETING_INFORMATION_it           ?	3_4_0	EXIST::FUNCTION:
 OSSL_GENERAL_NAMES_print                ?	3_4_0	EXIST::FUNCTION:
-d2i_ATTRIBUTES_SYNTAX                   ?	3_4_0	EXIST::FUNCTION:
-i2d_ATTRIBUTES_SYNTAX                   ?	3_4_0	EXIST::FUNCTION:
-ATTRIBUTES_SYNTAX_free                  ?	3_4_0	EXIST::FUNCTION:
-ATTRIBUTES_SYNTAX_new                   ?	3_4_0	EXIST::FUNCTION:
-ATTRIBUTES_SYNTAX_it                    ?	3_4_0	EXIST::FUNCTION:
+d2i_OSSL_ATTRIBUTES_SYNTAX              ?	3_4_0	EXIST::FUNCTION:
+i2d_OSSL_ATTRIBUTES_SYNTAX              ?	3_4_0	EXIST::FUNCTION:
+OSSL_ATTRIBUTES_SYNTAX_free             ?	3_4_0	EXIST::FUNCTION:
+OSSL_ATTRIBUTES_SYNTAX_new              ?	3_4_0	EXIST::FUNCTION:
+OSSL_ATTRIBUTES_SYNTAX_it               ?	3_4_0	EXIST::FUNCTION:
+d2i_OSSL_USER_NOTICE_SYNTAX             ?	3_4_0	EXIST::FUNCTION:
+i2d_OSSL_USER_NOTICE_SYNTAX             ?	3_4_0	EXIST::FUNCTION:
+OSSL_USER_NOTICE_SYNTAX_free            ?	3_4_0	EXIST::FUNCTION:
+OSSL_USER_NOTICE_SYNTAX_new             ?	3_4_0	EXIST::FUNCTION:
+OSSL_USER_NOTICE_SYNTAX_it              ?	3_4_0	EXIST::FUNCTION:
```

Comment:
do these need to have documentation added?

Reply:
They are meant to appear there because they are defined in public header files, so we need ordinals for them.  And I had missed that you added them to the pod file, I was looking for actual descriptions of what they do, but thats not needed for these functions I don't think